converter/dicom-cast/src/Microsoft.Health.DicomCast.Core.UnitTests/Features/Worker/DicomCastWorkerTests.cs

```diff
@@ -39,12 +43,18 @@ public DicomCastWorkerTests()
 
         _fhirService = Substitute.For<IFhirService>();
```

Comment:
Question for my understanding: why are _changeFeedProcessor and _hostApplication initialized outside of the constructor whereas _fhirService and _telemetryClient are initialized inside?

Reply:
It was like this before. I updated the code such that they are initialized at the same place